lib/oxigraph/Cargo.toml

```diff
@@ -13,10 +13,14 @@ description = "SPARQL database and RDF toolkit"
 edition.workspace = true
 rust-version.workspace = true
 
+[lib]
+crate-type = ["rlib", "cdylib"]
```

Comment:
not useful if you don't build the `oxigraph` crate as a dynamic library directly but an other crate having Oxigraph in it's dependencies. It does not apply in this case.